std/grpc/wire.go

```diff
@@ -35,9 +35,17 @@ func ProvideConn(ctx context.Context, req *Backend) (*grpc.ClientConn, error) {
 	return client.Dial(ctx, endpoint, grpc.WithTransportCredentials(insecure.NewCredentials())) ///  XXX mTLS etc.
 }
 
-func Loopback(ctx context.Context) (*grpc.ClientConn, error) {
+func Loopback(ctx context.Context, grpcOpts ...grpc.DialOption) (*grpc.ClientConn, error) {
 	endpoint := fmt.Sprintf("127.0.0.1:%d", server.ListenPort())
```

Comment:
BTW why don't we use `net.JoinHostPort` for these?

Reply:
I guess here is not that bad as we explicitly use IPv4 (and  `server.ListenPort()` is an integer and would need to be converted to string to be use in `net.JoinHostPort`).

Reply:
and from what I heard people do not like `strconv.Itoa()`

Reply:
changed btw

Reply:
Yeah it sucks that the port must be a string in `net.JoinHostPort`, I forgot about that.